app/src/main/java/com/example/journal_app/data/model/logIt.java

```diff
@@ -0,0 +1,58 @@
+package com.example.journal_app.data.model;
+import java.time.LocalDateTime; 
+import java.time.format.DateTimeFormatter;
+class LogIt
+{
+  String UserID;
+  String DeviceID;
+  String Output = "default"; // stores the value being put out to the log.
+  LocalDateTime RawDate = LocalDateTime.now(); //Date in LocalDateTime Format. Default is on creation of instance
+  String Date = "Default O'Clock"; //for the date format turned into a string.
+  //datetimeformatter is the format the RawDate is processed to when converting to string Date.
+  DateTimeFormatter Format = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+  
+  LogIt(String UserIDInput, String DeviceIDInput)
```

Comment:
public constructor so that we can access this package from outside the data.model package